docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -70,7 +82,7 @@ def events(context):
     )
 ```
 
-Alternatively, if you are using an I/O manager to handle saving and loading your data, you'll need to ensure the I/O manager is also using these methods. If you're using any of the built-in database I/O managers, like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb), you'll have this out-of-the-box.
+Alternatively, if you are using an I/O manager to handle saving and loading data, you'll need to ensure the I/O manager is also using these methods. If you're using any of the built-in database I/O managers, like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb), you'll have this out-of-the-box.
```

Comment:
Maybe we should add a note that file system I/O managers do not have these capabilities?